lib/Phpfastcache/Drivers/Mongodb/Config.php

```diff
@@ -195,4 +200,24 @@ public function setDatabaseName(string $databaseName): self
         $this->databaseName = $databaseName;
         return $this;
     }
+
+    /**
+     * @return array
+     */
+    public function getOptions(): array
+    {
+        return $this->options;
+    }
+
+    /**
+     * https://docs.mongodb.com/manual/reference/connection-string/#connections-connection-options
```

Comment:
Add phpDoc annotation "@see" before the link, thanks !

Reply:
Thank you for letting me know! I added @see to the comment!